File: php-laraval-app/app/Models/Country.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Country extends Model
{
    use HasFactory;

    /**
     * Indicates if the model should be timestamped.
     *
     * @var bool
     */
    public $timestamps = false;

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'code',
        'name',
        'currency',
        'dial_code',
        'flag',
    ];

    public function cities(): HasMany
    {
        return $this->hasMany(City::class);
    }
}

File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/Country.cs
namespace Downstairs.Infrastructure.Persistence.Models;

public partial class Country
{
    public long Id { get; set; }

    public string Code { get; set; } = null!;

    public string Name { get; set; } = null!;

    public string Currency { get; set; } = null!;

    public string DialCode { get; set; } = null!;

    public string Flag { get; set; } = null!;

    public virtual ICollection<City> Cities { get; set; } = new List<City>();
}